src/software/ai/hl/stp/play/enemy_ball_placement/enemy_ball_placement_play_fsm.cpp

```diff
@@ -0,0 +1,73 @@
+#include "software/ai/hl/stp/play/enemy_ball_placement/enemy_ball_placement_play_fsm.h"
+
+EnemyBallPlacementPlayFSM::EnemyBallPlacementPlayFSM(TbotsProto::AiConfig ai_config)
+    : ai_config(ai_config),
+      crease_defenders({std::make_shared<CreaseDefenderTactic>(
+                            ai_config.robot_navigation_obstacle_config()),
+                        std::make_shared<CreaseDefenderTactic>(
+                            ai_config.robot_navigation_obstacle_config())}),
+      move_tactics({
+          std::make_shared<MoveTactic>(),
+          std::make_shared<MoveTactic>(),
+          std::make_shared<MoveTactic>(),
+      }),
+      distance_to_keep(ENEMY_BALL_PLACEMENT_DISTANCE_METERS + 0.2)
```

Comment:
To be safe, it might be a good idea to increase this a bit further by also taking into account the robot radius